packages/mobile/src/send/utils.ts

```diff
@@ -126,7 +126,11 @@ export function showLimitReachedError(
   return showError(ErrorMessages.PAYMENT_LIMIT_REACHED, ALERT_BANNER_DURATION, translationParams)
 }
 
-export function* handleSendPaymentData(data: UriData, cachedRecipient?: RecipientWithContact) {
+export function* handleSendPaymentData(
+  data: UriData,
+  cachedRecipient?: RecipientWithContact,
+  isOutgoingPaymentRequest?: true
```

Comment:
should this be true by default?

Reply:
The type for `isOutgoingPaymentRequest` can only be `undefined` or `true` so I opted to just be a little more specific than using `boolean`.